main.js

```diff
@@ -0,0 +1,57 @@
+var boardArr = [];
+
+var board = {
+
+  initializeGame: function() {
+    boardArr = [ ['5','4','3','2','1'], [], [] ];
+    console.log('Game begin, board is:');
+    this.printBoard();
+  },
+
+  printBoard: function() {
+    var pegDisplay =  boardArr.map(function(peg) {
+      return ('--- '+peg.join(' '));
+     })
+     console.log(pegDisplay[0]);
+     console.log(pegDisplay[1]);
+     console.log(pegDisplay[2]);
+  },
+
+  moveDisc: function(fromPeg,toPeg) {
+    if(fromPeg < 1 || fromPeg > 3) {
```

Comment:
A switch statement would be cleaner and more reusable here, instead of so many if/else blocks.